### DISCUSSION TITLE
Explorer

### DISCUSSION THREAD

**Discussion by Mihaitafox11** [NillionNetwork/discussions#160] [General]
Hey,

I'm trying to build a Nillion Explorer, already got the domain (nillionscan.com) but I need a way to get the data from the blockchain.

From what I know, I can only get by running a node.

Is there any way I can apply for a node?

Any other solutions are welcomed, thanks!

**Comment by crypblizz8** [comment]
Currently not available at the moment. Will announce anything on our [twitter](https://x.com/nillionnetwork) in the future.
